test/fakes/fakeservercatalog/catalog.go

```diff
@@ -38,12 +38,12 @@ func (c *Catalog) DataStores() []*catalog.ManagedDataStore {
 	return c.dataStores
 }
 
-func (c *Catalog) SetNodeAttestors(nodeAttestors ...nodeattestor.NodeAttestor) {
+func (c *Catalog) SetNodeAttestors(baseName string, nodeAttestors ...nodeattestor.NodeAttestor) {
```

Comment:
this feels a little clunky to me... i'd think i'd rather leave the `Set` methods as they are and instead introduce a new method (which the Set method could use):

```
func (c *Catalog) SetNodeAttestors(nodeAttestors ...nodeattestor.NodeAttestor) {
    c.nodeAttestors = nil
    for i, nodeAttestor := range nodeAttestors {
        c.AddNodeAttestorNamed(pluginName("nodeattestor", i), nodeAttestor)
    }
}

func (c *Catalog) AddNodeAttestorNamed(name string, nodeAttestor nodeattestor.NodeAttestor) {
    	c.nodeAttestors = append(c.nodeAttestors, catalog.NewManagedNodeAttestor(
			nodeAttestor, common.PluginConfig{
				PluginName: name,
			}))
}
```

Reply:
Right, that looks better :+1: